src/sentry/migrations/0813_rm_alertruleactivations.py

```diff
@@ -0,0 +1,67 @@
+# Generated by Django 5.1.4 on 2025-01-10 20:05
+import django.db.models.deletion
+from django.db import migrations
+
+import sentry.db.models.fields.foreignkey
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.new_migrations.monkey.models import SafeDeleteModel
+from sentry.new_migrations.monkey.state import DeletionAction
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production.
+    # This should only be used for operations where it's safe to run the migration after your
+    # code has deployed. So this should not be used for most operations that alter the schema
+    # of a table.
+    # Here are some things that make sense to mark as post deployment:
+    # - Large data migrations. Typically we want these to be run manually so that they can be
+    #   monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   run this outside deployments so that we don't block them. Note that while adding an index
+    #   is a schema change, it's completely safe to run the operation after the code has deployed.
+    # Once deployed, run these manually via: https://develop.sentry.dev/database-migrations/#migration-deployment
+
+    is_post_deployment = False
+
+    dependencies = [
+        ("sentry", "0812_rm_activation_incident"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="alertruleactivationcondition",
+            name="alert_rule",
+            field=sentry.db.models.fields.foreignkey.FlexibleForeignKey(
+                db_constraint=False,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="activation_condition",
+                to="sentry.alertrule",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="alertruleactivations",
+            name="alert_rule",
+            field=sentry.db.models.fields.foreignkey.FlexibleForeignKey(
+                db_constraint=False,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="activations",
+                to="sentry.alertrule",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="alertruleactivations",
+            name="query_subscription",
+            field=sentry.db.models.fields.foreignkey.FlexibleForeignKey(
+                db_constraint=False,
+                null=True,
+                on_delete=django.db.models.deletion.SET_NULL,
+                to="sentry.querysubscription",
+            ),
+        ),
+        SafeDeleteModel(
+            name="AlertRuleActivationCondition", deletion_action=DeletionAction.MOVE_TO_PENDING
+        ),
+        SafeDeleteModel(
+            name="AlertRuleActivations", deletion_action=DeletionAction.MOVE_TO_PENDING
+        ),
+    ]
```

Comment:
Removing the models also generated this:

```
        migrations.RemoveField(
            model_name="alertruleactivations",
            name="alert_rule",
        ),
        migrations.RemoveField(
            model_name="alertruleactivations",
            name="query_subscription",
        ),
```

but I think this has happened in the past and we removed it without error. I was able to run the migration locally fine.